pennylane/io/io.py

```diff
@@ -20,15 +20,18 @@
 from functools import wraps
 from importlib import metadata
 from sys import version_info
-from typing import Any
+from typing import Any, overload
 
-from pennylane.wires import WiresLike  # pylint: disable=ungrouped-imports
+from pennylane.tape import QuantumScript
+from pennylane.transforms import convert_to_numpy_parameters
+from pennylane.wires import Wires, WiresLike
+from pennylane.workflow import QNode, construct_tape
 
 has_openqasm = True
 try:
     import openqasm3
 
-    from pennylane.io.qasm_interpreter import QasmInterpreter
+    from pennylane.io.qasm_interpreter import QasmInterpreter  # pylint: disable=ungrouped-imports
```

Comment:
```suggestion
    from pennylane.io.qasm_interpreter import QasmInterpreter
```
not that important, but let's see if this is actually necessary.

Reply:
Looks like it is necessary... 😮